src/machines/modelingMachine.ts

```diff
@@ -2316,7 +2317,21 @@ export const modelingMachine = setup({
 
         // Extract inputs
         const ast = kclManager.ast
-        const { selection, radius } = input
+        const { nodeToEdit, selection, radius } = input
+
+        // If this is an edit flow, first we're going to remove the old one
+        const pipeIndex = 5
+        if (
+          nodeToEdit &&
+          nodeToEdit[pipeIndex][0] &&
+          typeof nodeToEdit[pipeIndex][0] === 'number'
+        ) {
+          const r = locateExtrudeDeclarator(ast, nodeToEdit)
+          if (!err(r) && r.extrudeDeclarator.init.type === 'PipeExpression') {
+            r.extrudeDeclarator.init.body.splice(nodeToEdit[pipeIndex][0], 1)
+          }
+        }
+
```

Comment:
Okay so I'm guessing that this only works for editing fillets/chamfers that are in a pipeExpression?
Is that easy to fix for ones that are not in a pipe?, because while our code-mods might put them in a pipe, it would be good if the edit still worked without the pipe because either the user or AI might not put it in a pipe.

With that said this magic 5 makes me nervous, I'm guessing 5 is how deep in the editNode the pipeExpression is on the most common cases. I know we're trying to get the basic cases working because editing stuff within function definitions or in if statements is whole can of worms we don't the band width for right now, but maybe using 
`const pipeIndex = nodeToEdit.findIndex(([_, type]) => type === 'PipeExpression')` would make this a little more robust?